src/measure_theory/integral/circle_integral_transform.lean

```diff
@@ -0,0 +1,220 @@
+/-
+Copyright (c) 2022 Chris Birkbeck. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Birkbeck
+-/
+import analysis.complex.cauchy_integral
+import analysis.analytic.basic
+import analysis.calculus.parametric_interval_integral
+import data.complex.basic
+import measure_theory.integral.circle_integral
+/-!
+# Circle integral transform
+
+In this file we define the circle integral transform of a function `f` with complex domain. This is
+defined as $(2πi)^{-1}\frac{f(x)}{x-w}$ where `x` moves along a circle. We then prove some basic
```

Comment:
Here and elsewhere, you use the unicode character π inside $ $ signs. Is the code which compiles the manual smart enough to deal with that? (I don't know.)

Reply:
I don't actually know if it is, but I expect so, since I've seen in in other files, such as `circle_integral.lean`

Reply:
[Looks like you're right](https://leanprover-community.github.io/mathlib_docs/measure_theory/integral/circle_integral.html#circle_integral.integral_sub_inv_of_mem_ball). Feel free to mark this one as resolved.